example/bond/README.md

```diff
@@ -15,6 +15,14 @@ npm install
 npm run compile
 ```
 
+Build the [typescript sdk](../../sdk/typescript) which is a dependency:
+
+```shell
+cd ../../sdk/typescript
+npm install
+npm run build
+```
```

Comment:
I also added this note in https://github.com/LF-Decentralized-Trust-labs/paladin/pull/389